addons/website_sale/views/website_sale_delivery_templates.xml

```diff
@@ -18,7 +18,7 @@
     <template id="payment_delivery_methods">
         <input t-att-value="delivery.id" t-att-id="'delivery_%i' % delivery.id" t-att-delivery_type="delivery.delivery_type" type="radio" name="delivery_type" t-att-checked="order.carrier_id and order.carrier_id.id == delivery.id and 'checked' or False"/>
         <label class="label-optional" t-field="delivery.name"/>
-        <t t-set='badge_class' t-value="'float-end ' + 'badge text-bg-secondary'" />
+        <t t-set='badge_class' t-value="'float-end fw-bold'" />
```

Comment:
To define a variable was needed by previous implementations (see https://github.com/odoo/odoo/pull/105848/files#diff-83a28c43e60e845a7eb01351dd94670014c0aedc760d28b1d5dac8f6a0610fbaL21) but it doesn't seems the case anymore.

I think that `badge_class` can be removed and its values added to the following `<span>` directly.